cogboard-webapp/src/components/widgets/types/JiraBucketWidget.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { array } from 'prop-types';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableCell from '@material-ui/core/TableCell';
+import TableBody from '@material-ui/core/TableBody';
+
+const JiraBucketWidget = ({ buckets }) => {
+  return (
+    <>
+      <div>
```

Comment:
What's the point of this div?  Is it used for styling somehow? If it is, why doesn't it have a class? It seems to defeat the purpose of using the React Fragment in the preceding line.

Reply:
it is used for styling now.